tests/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ function(add_unit_test name)
   if(WITH_BLOCKS_RUNTIME)
     target_link_libraries(${name} PRIVATE BlocksRuntime)
   endif()
-  if(BSD_OVERLAY_FOUND)
+  if(BSD_OVERLAY_FOUND AND (NOT ${name} MATCHES "dispatch_c99"))
```

Comment:
These exclusions are ugly.  How about adding a `NO_BSD_OVERLAY` keyword option to the `add_unit_test` so that it is generic.